treeherder/intermittents_commenter/constants.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
```

Comment:
This logger is unused

Reply:
It is being used - it's being imported to commenter.py along with all of the constants in that file.

Reply:
Ah ok. In that case, it must be moved to the other file. `logger` isn't a constant, and it's best practice to call `getLogger()` once per file, since `__name__` will resolve to the name of the current filename, so otherwise the `logger.{info,warning,...}(...)` calls will have the wrong filename shown as a prefix in the logs.

Reply:
> it's being imported to commenter.py along with all of the constants in that file

(This is also another reason why wildcard imports are not recommended - it hides what is being imported, and makes it harder to see what's used/unused)

Reply:
Good point :) I've cleaned up that file and the imports.